nexus-sdk-js/src/Project.ts

```diff
@@ -96,16 +96,6 @@ export default class Project {
   }
 
   async getResource(id: string): Promise<Resource> {
-    try {
-      const resourceResponse: ResourceResponse = await httpGet(id, false);
-      const resource = new Resource(
-        this.orgLabel,
-        this.label,
-        resourceResponse,
-      );
-      return resource;
-    } catch (e) {
-      return e;
-    }
+    return await getResource(id, this.orgLabel, this.label);
```

Comment:
does `getResource` always return a resource?

Reply:
It does, or throws an error